.github/workflows/main.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
+  lint:
     runs-on: ubuntu-latest
```

Comment:
Is it possible to run the lint command on one element of the build matrix instead of building an entire venv just for lint ?
It could also run on all cases of the build matrix, as the pylama and mypy runs are quick

Reply:
Did not find a way to run lint for only one version of the matrix. If you prefer we can run linter for each version